osgar/tools/lidarview.py

```diff
@@ -223,7 +223,7 @@ def get_image(data):
     elif isinstance(data, tuple):
         img_data, depth_data = data
         image = pygame.image.load(io.BytesIO(img_data), 'JPG').convert()
-        g_depth = decompress_depth(depth_data)
+        g_depth = decompress_depth(depth_data)/1000
```

Comment:
This is incompatible with virtual subt logs because there are different units (mm versus m). I think the mm (int16) are better choice . Or not?

Reply:
this is breaking change so I do not like it ... I actually do not like the RGBD bundle, but I understand it from "synchronized source" point of view. Yes it is not practical and the data are bigger (twice?)